src/MigrationLog.php

```diff
@@ -40,7 +42,7 @@ public static function from_wp_options(): self {
 	 */
 	public function already_ran( string $migration_id ): bool {
 		foreach ( $this->done_migrations as $migration ) {
-			if ( $migration['id'] === $migration_id ) {
+			if ( $migration['id'] === $migration_id && $migration['success'] ) {
```

Comment:
We might want to look into allowing (maybe enforcing) some kind of success query, so that we can actually determine whether the script needs to run for any given data.

Reply:
Though that might be a bit overkill for this PR, all I wanted to do initially was make an option default to true :smile: